docs/inputs/index.md

```diff
@@ -102,8 +102,13 @@ Let's go with some example files to demonstrate Hjem's linking capabilities.
    ```nix
    ".config/bar".source = pkgs.writeTextFile "file-foo" "file contents";
    ```
+   Note : `.source` also supports directly provided path.
+   ```nix
+   ".config/bar".source = ./foo;
+   ```
```

Comment:
```suggestion
   ::: {.note}
   
   `.source` attribute also also supports passing paths directly:
   
   ```nix
   ".config/bar".source = ./foo;
   ```

   :::
```

You can use the admonitions feature of our documentation generator